project/src/main/world/glowy-sphere.gd

```diff
@@ -5,3 +5,4 @@ Script for showing a placeholder chattable object in the overworld.
 
 func _ready() -> void:
 	set_meta("chat_path", "res://assets/main/dialog/glowy-sphere.json")
+	set_meta("chat_bubble_type", ChatIcon.THOUGHT)
```

Comment:
This paradigm shift is worth documenting in the commit comment.

Reply:
Expanded commit comment; `Creatures no longer have a chat icon -- instead, a separate ChatIcons node
generates chat icons for all creatures, and anything else that is in the
chattables group. Instead of customizing the chat icon by changing the
chat icon's properties, nodes can define metadata which affects their chat
icon appearance, such as the 'chat_bubble_type' metadata`